src/resmom/linux/mom_mach.c

```diff
@@ -129,13 +129,16 @@ static char	procfs[] = "/proc";
 static DIR	*pdir = NULL;
 static int	pagesize;
 static long	hz;
+#if	MOM_CPUSET
+static char cpusetfs[] = "/dev/cpuset/PBSPro";
```

Comment:
Fixed it. We don't need to import mom_mach.h again as it is included under mom_func.h